### PR TITLE
Fix of CI compilation steps in SM 1.11

### PR DESCRIPTION
**Pull request by domikuss**
I present my Pull with fixes for correct compilation on SM 1.11 (and SM 1.10)
- The Hosties and LastRequest library has been moved to the main repository due to incorrect code in the developer's repository
- Now is used the library from the unofficial SteamWorks repository for compilation, because there have been changes in SM 1.11 that no longer support the old syntax
- In the sdkhooks_voice library in SM 1.11, forwards with identical voiceannounce_ex functionality were added. Therefore, it was decided to use standard methods for SM 1.11, and for SM 1.10 as before. **(Module - [JWP] Be quiet)**
- Other important fixes

### CODE REVIEW

File: addons/sourcemod/scripting/jwp_bequiet.sp [hunk 1]
```diff
@@ -40,6 +43,28 @@ public void OnClientSpeakingEx(client)
 		}
 	}
 }
+#else
+public void OnClientSpeaking(client)
```
Comment (TiBarification) [inline addons/sourcemod/scripting/jwp_bequiet.sp:28]: code is duplicated can u try to use like
```cpp
#if SOURCEMOD_V_MINOR == 10
public void OnClientSpeakingEx(client)
#else
public void OnClientSpeaking(client)
#endif
{
 // do stuff
}
```

Or moving that part to internal function like `_OnClientSpeaking(int client)`

File: addons/sourcemod/scripting/jwp_mute.sp [hunk 2]
```diff
@@ -50,6 +50,8 @@ public Action Event_OnRoundEnd(Event event, const char[] name, bool dontBroadcas
 			}
 		}
 	}
+
+	return Plugin_Continue;
```
Comment (TiBarification) [inline addons/sourcemod/scripting/jwp_mute.sp]: Do we really need it there?
There might be optimization, callback might return nothing, so we can replace **Action** to **void**

File: addons/sourcemod/scripting/include/hosties.inc [hunk 3]
```diff
@@ -0,0 +1,159 @@
+/*
```
Comment (TiBarification) [inline addons/sourcemod/scripting/include/hosties.inc]: do we really need too have a dupe of this file , do u know where it exist on some another repo?
Same for lastrequest.inc

**Review by TiBarification — COMMENTED** [review]
Great work, but

File: addons/sourcemod/scripting/jwp_bequiet.sp (hunk 1 above)
Comment (domikuss) [reply addons/sourcemod/scripting/jwp_bequiet.sp:28]: Fixed.

File: addons/sourcemod/scripting/jwp_mute.sp (hunk 2 above)
Comment (domikuss) [reply addons/sourcemod/scripting/jwp_mute.sp]: Oh.. Night fixes are not the best thing. Fixed.

**Review by TiBarification — APPROVED** [review]

**Pull request merged**